app/shared/models/notification.rb

```diff
@@ -69,13 +74,15 @@ class Notification
     # @param  [String] user_id
     # @param  [String] name
     # @param  [String] message
-    def initialize(id: nil, priority: nil, type: nil, user_id: nil, name: nil, message: nil, created_at: nil, updated_at: nil)
+    # @param  [String] details
+    def initialize(id: nil, priority: nil, type: nil, user_id: nil, name: nil, message: nil, created_at: nil, updated_at: nil, details: nil)
```

Comment:
Do you think we could change the rubocop max line length from [370](https://github.com/fastlane/ci/blob/master/.rubocop.yml#L174) to something like 120?

Reply:
Yeah, totally. I think 120 is perfect 👍

Reply:
(feel free to open an issue for that!)

Reply:
Sounds good! Will do!

Reply:
Updated in #506